### ISSUE TITLE
Flower does not work in docker-compose anymore

### ISSUE THREAD

**Issue by jLebioda** [elixir-luxembourg/daisy#197]
**Describe the bug (current behaviour)**

**To reproduce**

1. Run `docker-compose up`
2. Notice in the output:

```flower_1  | ImportError: cannot import name 'Command' from 'celery.bin.base' (/usr/local/lib/python3.9/site-packages/celery/bin/base.py)
daisy_flower_1 exited with code 1
```

**Expected behavior**
Flower starts normally

**Environment:**

 - Deployment: dev deployment
 - Version 1.4
 
**Additional information (error logs, screenshots)**
probably due to a recent update in flower

**Comment by jLebioda** [comment]
For an easy fix, just pinpoint the _flower's_ version (`docker-compose.yaml:71` ):
```
69. # task monitoring
70. flower:
71.      image: mher/flower
```

to

```
71.    image: mher/flower:0.9.5
```

It'd be good to investigate what changed in the new version, too

**Comment by jLebioda** [comment]
This has been fixed in https://github.com/elixir-luxembourg/daisy/commit/04e3a76437d56211ee356159377e7ba3a997c50a;
however we should also look into why the new version of flower was not working and fix it (probably we'd need to update Celery)

**Comment by jLebioda** [comment]
Will be fixed once #321 is merged
